python_modules/dagster/dagster/_utils/container.py

```diff
@@ -5,6 +5,10 @@
 
 from dagster._time import get_current_timestamp
 
+# When cgroup memory is not constrained, the limit value will be a high value close to 2^63 - 1,
+# Ex: AWS ECS returns 2^63 - 2^10 = 9223372036854000000
+UNCONSTRAINED_CGROUP_MEMORY_LIMIT = 9223372036854000000
```

Comment:
actually I wonder if that value isn't an artefact of json arbitrary precision

Reply:
also, why not in internal since I am not using yet in Dagster - 

I felt it's the appropriate place to put it with the container utilization metrics utils + I think I will reuse this to catch the same edge cases in run metrics.